boskos/handlers/handlers.go

```diff
@@ -277,15 +279,14 @@ func handleReset(r *ranch.Ranch) http.HandlerFunc {
 
 		expire, err := time.ParseDuration(expireStr)
 		if err != nil {
-			logrus.WithError(err).Errorf("Invalid expiration: %v", expireStr)
+			logrus.WithError(err).Debugf("Invalid expiration: %v", expireStr)
```

Comment:
should this have been wrapped as a badRequestError too?

similarly for the case above (where fields are missing)